Examples/Swift/FillPatternExample.swift

```diff
@@ -0,0 +1,59 @@
+import Mapbox
+
+@objc(FillPatternExample_Swift)
+
+class FillPattern_Swift: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Set the map’s size, style, center coordinate, and zoom level.
+        let mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.styleURL = MGLStyle.darkStyleURL(withVersion: 9)
```

Comment:
Is this example dependent on any implementation details in v9? If not and it’s purely additive, then we can use `MGLStyle.darkStyleURL()`.